ui/app/models/policy.js

```diff
@@ -4,7 +4,6 @@ import { computed } from '@ember/object';
 import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
 
 export default Model.extend({
-  errors: attr('array'),
```

Comment:
`errors` is an internal property on the model class and overriding it was throwing errors on initial load of the policy model.

Reply:
I had added [this](https://github.com/hashicorp/vault/pull/14551/files) because glimmerizing `MessageError` for some reason no longer triggered error messages, so I manually added it to the component. 

Definitely makes sense that overriding would cause issues -and in the `policy/create.hbs` file from the linked PR above, `this.model.errors` should still behave, right?

Reply:
Yes I tested and the error message is still displayed. The `errors` property is set by the Adapter when there is a network request failure.